packages/dialtone-vue3/components/rich_text_editor/extensions/emoji/emoji.js

```diff
@@ -40,19 +34,16 @@ const shortCodePasteMatch = (text) => {
 };
 
 export const Emoji = Node.create({
+  name: 'emoji',
   addOptions () {
     return {
       HTMLAttributes: {},
-      suggestion: {
-        char: ':',
-        pluginKey: EmojiPluginKey,
-      },
     };
   },
-  name: 'emoji',
   group: 'inline',
   inline: true,
-  selectable: true,
+  selectable: false,
```

Comment:
Shouldn't this be selectable? I notice if I try to highlight when an emoji is at the end of the text it doesn't actually highlight anything, though not sure if making it `selectable: true` will even make a difference...

![2024-09-05 09 47 00](https://github.com/user-attachments/assets/3698b308-840c-41ee-8a93-df0889bb8207)

Reply:
Must be something related to the rich-text-editor besides the emoji, you can actually select it with the keyboard but looks like rich-text-editor is trying to remove some spacing at the end while clicking.

The "selectable" option there is to avoid putting the cursor behind the emojis.

Reply:
![2024-09-05 12 40 39](https://github.com/user-attachments/assets/9df48e59-26f2-4bb8-83f0-0a89ca59fd6f)

Exactly the same issue with mentions.

Reply:
Updating `@tiptap/*` dependencies to latests fixed the issues.

Reply:
nice! I'm happy it was that easy